pkgs/by-name/uv/uv-sort/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchPypi,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "uv-sort";
+  version = "0.5.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "uv_sort";
+    inherit version;
+    hash = "sha256-qCShDuKBFS4omcsntZ1wzRtAKTbp8CfTw0IIAgxBvcE=";
+  };
```

Comment:
fetchFromGitHub is preferred unless there is a compelling reason to not use it. If so, put in a comment

Reply:
My comment [here](https://github.com/NixOS/nixpkgs/pull/388382#issuecomment-2708857805) shows that a build from GitHub does not work.